src/Compilers/Core/Portable/Emit/EditAndContinue/DefinitionMap.cs

```diff
@@ -690,7 +692,7 @@ static ImmutableArray<string> getHoistedVariableNames(ImmutableArray<ISymbolInte
             {
                 provider = Baseline.DebugInformationProvider(MetadataTokens.MethodDefinitionHandle(methodRowId));
             }
-            catch (Exception e) when (e is InvalidDataException or IOException)
+            catch (Exception e) when (e is InvalidDataException or IOException or BadImageFormatException)
```

Comment:
>catch (Exception e) when (e is InvalidDataException or IOException or BadImageFormatException) [](http://example.com/codeflow?start=12&length=94)

We don't want to report anything for this `catch`?

Reply:
It's not a critical error. If a method is deleted and its body is not updated to throw an exception it would execute the original IL. Perhaps we could add a new warning but I don't think it's worth the effort.